theme/themes/pxt/globals/site.variables

```diff
@@ -36,10 +39,10 @@
 @importGoogleFonts: false;
 
 @docsHeaderFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
-@docsPageFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
+@docsPageFont: "Times New Roman", serif;
```

Comment:
really "Times New Roman"?

Reply:
It is mainly for descriptions in card. It doesn't look too bad. We can discuss as well.

![image](https://user-images.githubusercontent.com/704682/59700690-7c2fde80-91b9-11e9-8d2e-a2ce8696a7cb.png)

Reply:
(I talked to Arturo) We will switch to Helvetica, Arial